src/SuperSelectField.js

```diff
@@ -146,7 +151,7 @@ class SelectField extends Component {
       const updatedValues = selectedItemExists
         ? selectedItems.filter(obj => !areEqual(obj.value, selectedItem.value))
         : selectedItems.concat(selectedItem)
-      this.setState({ selectedItems: updatedValues })
+      this.setState({ selectedItems: updatedValues }, this.getSelected())
```

Comment:
If for some reason the `setState` here fails, then `getSelected()` will not be called.